12.8-2.3/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM postgres:12.8
+
+ENV PLV8_VERSION=2.3.15 \
```

Comment:
You can just use `VAR` for this, no reason for this to be an environment variable.